packages/contracts/contracts/optimistic-ethereum/OVM/predeploys/OVM_GasPriceOracle.sol

```diff
@@ -46,15 +47,34 @@ contract OVM_GasPriceOracle is Ownable {
      ********************/
 
     /**
-     * Allows the owner to modify the l2 gas price.
+     * A getter for the L2 gas price.
+     */
+    function getGasPrice()
+        public
+        view
+        override
+        returns (
+            uint256 _gasPrice
```

Comment:
We prefer not having a named return value because it can lead to errors down the line. Probably wouldn't be an issue here but better to be consistent.